### ISSUE TITLE
Not getting android notifications

### ISSUE THREAD

**Issue by jimfulton** [gitterHQ/gitter#1479]
On either of 2 devices I installed the app on yesterday:

- Note 2 Android 4.2.2, Gitter 3.0.0

- ASUS Zen Pad, Android 6.0, Gitter 3.0.0

Notifications aren't blocked.

**Comment by trevorah** [comment]
Hi @jimfulton, we dont send notifications if you have the web or desktop client running. Do you have them open?

**Comment by jimfulton** [comment]
I might have had them open but inactive (e.g. away from my machine and screen saver likely on).

I have definitely missed messages.

Are these rules documented somewhere?  If not, I guess I'll try some experiments.

**Comment by mojaveazure** [comment]
@trevorah  Android notifications don't come through if my computer is asleep with Gitter open. This ends up with me missing a bunch of messages. Can this be changed, or turned into an option for those that are fine with notification spam?

**Comment by kmscode** [comment]
I'm also having problems with Android notifications.

Currently running Gitter 3.1.1 on a Google Pixel running 7.1.1 (January security update).

Gitter is definitely not open on my desktop or in the browser. I've been having this problem for some time now and opened some support requests (most recently 3832, but previously 3391 on October 13th prior to switching devices).

This is more of a problem now since due to #1570 I can't even rely on e-mails to get notifications about messages that I may have missed when not logged into a client on my desktop.

I would prefer the Android notifications over e-mail notifications.

**Comment by mydigitalself** [comment]
@kmscode are you getting these notifications now? I just sent you a test message via Gitter.

**Comment by kmscode** [comment]
@mydigitalself received. Thanks for getting this worked out! Much appreciated!!! (email was received as well)

**Comment by MadLittleMods** [comment]
@jimfulton @mojaveazure Are you still running into issues?

If you are still having issues, try the following. If you are up to dump the app data for us to investigate, try this before: http://stackoverflow.com/a/17876731/796832

> The fix is to clear the app data in settings.
>
>@pitaj, https://github.com/gitterHQ/gitter/issues/1289#issuecomment-225465446

**Comment by mojaveazure** [comment]
Issue is still there. I now get email notifications, where I didn't before. There was a brief period of time where I would get Android notifications, but now it seems like the app sends notifications when it feels like it. Basically, I'll get a notification for every third or fifth message. I'll see if I can dump the app data.

Oh, and clearing the app data didn't help.

**Comment by kmscode** [comment]
Yeah. I havet gotten android notifications since my previous comment on this issue. Don't know when it stopped working because I don't get (or expect to get) messages daily... All the more reason why the notifications are important to me. Thanks!
